subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,80 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### is there unit tests for each functionality and are the test run for each PR.
```

Comment:
The subject should reflect this request - as it is right now, tests seem to be only for bonus points. It might be better to specify that unit tests are mandatory